tensorflow/compiler/xla/service/cpu/ir_emitter.cc

```diff
@@ -506,7 +506,7 @@ Status IrEmitter::HandleReduceWindow(HloInstruction* reduce_window,
 
         llvm_ir::IrArray::Index input_index(index.size());
         llvm::Value* in_bounds_condition = nullptr;
-        for (int64 i = 0; i < index.size(); ++i) {
+        for (auto i = 0; i < index.size(); ++i) {
```

Comment:
curious - I was going to ask earlier if "auto" would work for some earlier cases, but didn't think it would (I assumed it would derive the type from the i=0 assignment).  Do you know what the auto type here is?  I guess "int"?